membership_withdrawl/__openerp__.py

```diff
@@ -0,0 +1,40 @@
+# -*- encoding: utf-8 -*-
+# Python source code encoding : https://www.python.org/dev/peps/pep-0263/
+##############################################################################
+#
+#    OpenERP, Odoo Source Management Solution
+#    Copyright (c) 2015 Antiun Ingeniería S.L. (http://www.antiun.com)
+#                       Antonio Espinosa <antonioea@antiun.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': "Membership withdrawal",
+    'category': 'Association',
+    'version': '1.0',
+    'depends': [
+        'membership',
```

Comment:
Yeah, I see. Well, you can practically reference the parent view instead the membership one, losing only the "section" to put the field into. As you like, but the other approach can be applied to other things, not just membership, but partnerl withdrawal in general.